apps/userdashboard/views.py

```diff
@@ -65,10 +67,19 @@ def actions(self):
                 ContentType.objects.get_for_model(Chapter)
             ]
         )
-        return [action for action in comment_actions if
-                not action.obj.is_blocked
-                and action.target.creator == user
-                and action.actor != user]
+        filtered_comment_actions = [action for action in comment_actions if
+                                    not action.obj.is_blocked
+                                    and action.target.creator == user
+                                    and action.actor != user]
+        statement_actions = Action.objects.filter(
+            obj_content_type=ContentType.objects.get_for_model(
+                ModeratorCommentStatement),
+        )
+        filtered_statement_actions = [action for action in statement_actions if
+                                      action.obj.comment.creator == user
+                                      and action.actor != user]
+        return sorted(filtered_comment_actions + filtered_statement_actions,
+                      key=lambda action: action.timestamp, reverse=True)
```

Comment:
This is somewhat complicated, but did not know how else to do this as the actions have different properties depending on the obj_content_type..